src/scripts/main.js

```diff
@@ -1,7 +1,28 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const body = document.body;
+  const container = document.createElement('div');
+  const h2 = document.createElement('h2');
+  const p = document.createElement('p');
+
+  container.className = `notification ${type}`;
+  container.style.top = `${posTop}px`;
+  container.style.right = `${posRight}px`;
+  body.appendChild(container);
+
+  h2.className = 'title';
+  h2.style.whiteSpace = 'nowrap';
+  h2.innerText = `${title}`;
+  container.appendChild(h2);
+
+  p.innerText = `${description}`;
+  container.appendChild(p);
+
+  setTimeout(() => {
+    // body.removeChild(container);
```

Comment:
The line to remove the notification from the DOM is commented out. If the intention is to hide the notification after a timeout, consider uncommenting this line or ensuring that the notification is properly removed from the DOM after it is hidden.